e2etests/test/testUtils.js

```diff
@@ -4,12 +4,12 @@
 'use strict';
 var deviceSdk = require('azure-iot-device');
 
-function createDeviceClient(deviceTransport, provisionedDevice) {
+function createDeviceClient(deviceTransport, provisionedDevice, modelId) {
```

Comment:
>provisionedDevice [](http://example.com/codeflow?start=45&length=17)

I should know this better, but if you're using DPS to create the device, the modelId goes into the provisioning payload somewhere.  I think this is for IoTCentral, so it may not be strictly 100% necessary, but, we should make sure we do in production code, and we should make sure our E2E tests are doing this too.  (At least I _think_ our E2E tests should do this.)

Reply:
I don't think we do use DPS, actually. We just add the device to the device registry.

Reply:
`provisionedDevice` is poor naming, I guess. I'll change it.

Reply:
Done